### ISSUE TITLE
`help` builtin crashes OSH when argument is passed

### ISSUE THREAD

**Issue by timetoplatypus** [oilshell/oil#51]
Any time I pass an argument to the `help` builtin, OSH crashes. It's also a bit odd because it explicitly has "/home/andy" in the error message.

```
$>   # I'm in bash here
$> osh --version
Oil version 0.2.0
Release Date: 2017-11-09 05:50:32+00:00
Arch: x86_64
OS: Linux
Platform: #1 SMP PREEMPT Wed Nov 8 11:54:06 CET 2017
Compiler: GCC 7.2.0
Interpreter: OVM
Interpreter version: 2.7.13
$>   # Dropping into OSH now
$> osh
$osh> help
Usage:
  help <topic>   -- show help on a given topic
  help toc       -- list help topics
  help osh-usage -- same as osh --help
  help oil-usage -- same as oil --help

View on the web: http://www.oilshell.org/$VERSION/doc/osh-quick-ref.html

$osh> help echo
Traceback (most recent call last):
  File "/home/andy/git/oilshell/oil/Python-2.7.13/Lib/runpy.py", line 174, in _run_module_as_main
  File "/home/andy/git/oilshell/oil/Python-2.7.13/Lib/runpy.py", line 72, in _run_code
  File "/home/andy/git/oilshell/oil/bin/oil.py", line 475, in <module>
  File "/home/andy/git/oilshell/oil/bin/oil.py", line 458, in main
  File "/home/andy/git/oilshell/oil/bin/oil.py", line 441, in OilMain
  File "/home/andy/git/oilshell/oil/bin/oil.py", line 329, in OshMain
  File "/home/andy/git/oilshell/oil/bin/oil.py", line 131, in InteractiveLoop
  File "/home/andy/git/oilshell/oil/bin/../core/cmd_exec.py", line 969, in Execute
  File "/home/andy/git/oilshell/oil/bin/../core/cmd_exec.py", line 933, in _Execute
  File "/home/andy/git/oilshell/oil/bin/../core/cmd_exec.py", line 743, in _Dispatch
  File "/home/andy/git/oilshell/oil/bin/../core/cmd_exec.py", line 962, in _ExecuteList
  File "/home/andy/git/oilshell/oil/bin/../core/cmd_exec.py", line 933, in _Execute
  File "/home/andy/git/oilshell/oil/bin/../core/cmd_exec.py", line 623, in _Dispatch
  File "/home/andy/git/oilshell/oil/bin/../core/cmd_exec.py", line 511, in _RunSimpleCommand
  File "/home/andy/git/oilshell/oil/bin/../core/cmd_exec.py", line 312, in _RunBuiltin
  File "/home/andy/git/oil/bin/../core/builtin.py", line 866, in Help
KeyError: 'echo'
FATAL: couldn't import from app bundle '/home/timetoplatypus/Documents/archBuilds/osh/pkg/osh/usr/bin/oil.ovm' (1)
Stripping the oil.ovm binary may cause this error.
See https://github.com/oilshell/oil/issues/47
$>   # OSH has crashed and I'm back in bash here
```

**Comment by andychu** [comment]
Oops, thanks for catching this.  Just fixed it.

If you pull the master branch you should be able to verify this.

**Comment by andychu** [comment]
Oh, and Python's zipimporter prints the original paths, not the paths from the .zip file, which is indeed weird.

It is because I compiled bytecode into the zip files, and the bytecode files have those paths.

I think that eventually we will move away from this app bundle format, but it's serving its purpose for now.  Although actually I can probably fix it with the "OPy compiler", but that probably won't be turned on for a few months.

**Comment by timetoplatypus** [comment]
Just updated my OSH installation to include the latest commits. I can verify that the `help` builtin is no longer crashing when passed a bad argument.

**Comment by andychu** [comment]
Fix released with 0.3.
